### PR TITLE
XW-3935 | enable MCF for non-english wiki

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3935
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Recirculation/RecirculationController.class.php [hunk 1]
```diff
@@ -76,7 +73,7 @@ public function footer() {
 		global $wgSitename, $wgCityId, $wgLanguageCode;
 
 		// Language code check is temporary to supress MCF for old discussions on non-en wikis;
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/RecirculationController.class.php:73]: MAJOR Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecirculationController.class.php#L20](https://github.com/Wikia/app/blob/b888fd60c7501bfdd17e2cb76f80ffc2b5474e4b/extensions/wikia/Recirculation/RecirculationController.class.php#L20): Remove this unused "$wgLanguageCode" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by rybmat — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by hakubo — APPROVED** [review]

File: extensions/wikia/Recirculation/RecirculationController.class.php [hunk 2]
```diff
@@ -87,12 +82,26 @@ public function footer() {
...
+		// we do not show n&s articles on non-english wikis
+		if ( $wgLanguageCode !== 'en' ) {
+			$numberOfWikiArticles = 11;
+			$numberOfNSArticles = 0;
+		}
+
 		if ( !$canShowDiscussions ) {
 			$numberOfWikiArticles ++;
-			$numberOfNSArticles ++;
+			if ( $wgLanguageCode === 'en' ) {
```
Comment (bkoval) [inline extensions/wikia/Recirculation/RecirculationController.class.php:93]: Seems like this is a repeatable logic - could we extract it to a method?

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 3]
```diff
@@ -56,64 +56,88 @@ require([
...
-			}).then(function () {
-				liftigniter.setupTracking(view.nsItemsSelector, mixedContentFooter.nsItems);
-				liftigniter.setupTracking(view.wikiItemsSelector, mixedContentFooter.wikiItems);
+	function prepareInternationalRecirculation () {
+		var mixedContentFooterData = [
+			liftigniter.prepare(mixedContentFooter.wikiItems),
+			discussions.prepare()
+		];
+		$.when.apply($, mixedContentFooterData).done(function (wikiItems, discussions) {
+			$mixedContentFooterContent.show();
```
Comment (bkoval) [inline extensions/wikia/Recirculation/js/recirculation.js:100]: This looks pretty similar to https://github.com/Wikia/app/pull/13670/files#diff-c7077e2816c4f2e1464b8533b6fcc3cfR79 -> Could we create a reusable method that takes a callback and render params?

**Review by bkoval — COMMENTED** [review]
Lookin' good, just two comments.

**Pull request merged**